packages/session-replay-browser/src/typings/session-replay.ts

```diff
@@ -39,14 +39,15 @@ export interface SessionReplayConfig extends Config {
   logLevel: LogLevel;
   flushMaxRetries: number;
   sampleRate: number;
+  sessionReplayId?: string;
 }
 
 export type SessionReplayOptions = Omit<Partial<SessionReplayConfig>, 'apiKey'>;
 
 export interface AmplitudeSessionReplay {
   init: (apiKey: string, options: SessionReplayOptions) => AmplitudeReturn<void>;
-  setSessionId: (sessionId: number) => void;
-  getSessionRecordingProperties: () => { [key: string]: boolean };
-  getSessionReplayProperties: () => { [key: string]: boolean };
+  setSessionId: ({ sessionId, sessionReplayId }: { sessionId?: number; sessionReplayId?: string }) => void;
+  getSessionRecordingProperties: () => { [key: string]: boolean | string | null };
```

Comment:
the legal requirement is to remove any reference to "getSessionRecordingProperties", we should do that as part of this work. I think we've had this deprecated for a while now.